src/app/screens/Record/styles.module.scss

```diff
@@ -33,28 +33,74 @@
   display: flex;
   flex-direction: column;
   flex-wrap: nowrap;
-  min-height: 3px; 
+  min-height: 3px;
 }
 
 .operator {
+  align-items: center;
   appearance: none;
-  border: 0;
   display: flex;
   flex: 0 0 100%;
-  flex-direction: row-reverse;
-  padding: 15px;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-right: 15px;
   width: 100%;
+
+  & button {
+    appearance: none;
+    background-color: $input-primary;
+    border: 0;
+    border-radius: 0;
+    font-size: 26px;
+    font-weight: 500;
+    height: 60px;
+    width: 60px;
+  }
 }
 
-.recToCalcdButton {
-  align-items: flex-start;
-  appearance: none;
+.recordButtons {
+  align-items: stretch;
   background-color: $gray;
   border: 0;
   color: $disabled-gray;
   display: flex;
-  font-size: 20px;
+  font-size: 16px;
   font-weight: 300;
-  padding: 12px;
+  justify-content: space-between;
   width: 100%;
 }
+
+.clear {
+  appearance: none;
+  background-color: $gray;
+  border: 0;
+  border-radius: 0;
+  color: $disabled-gray;
+  font-size: 16px;
+  font-weight: 300;
+  padding: 10px;
+  text-align: left;
+  width: 50%;
+
+  &:hover {
+    background-color: $black;
+  }
+}
+
+.recToCalcButton {
+  appearance: none;
+  background-color: $gray;
+  border: 0;
+  border-radius: 0;
+  color: $disabled-gray;
+  font-size: 16px;
+  font-weight: 300;
+  padding: 10px;
+  padding-right: 15px;
+  text-align: right;
+  width: 50%;
+
+  &:hover {
+    background-color: $black;
+  }
+}
```

Comment:
Extarct all the common styles props into another class and then extend it whenever you need it. If you have any doubt let me know!